test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -1024,6 +1104,17 @@ public static String getTestTransportType() {
         return useNio ? MockNioTransportPlugin.class : MockTcpTransportPlugin.class;
     }
 
+    private static boolean useZen2;
+
+    @BeforeClass
+    public static void setUseZen2() throws Exception {
+        useZen2 = false; // TODO: change to randomBoolean() later
+    }
+
+    public static boolean getUseZen2() {
+        return useZen2;
```

Comment:
I think this mechanism could just return `false` here and avoid introducing the field and (unused) setter yet.